JitsiConference.js

```diff
@@ -968,24 +1147,9 @@ JitsiConference.prototype.onCallEnded
         this.statistics.stopCallStats();
     }
     // Current JingleSession is invalid so set it to null on the room
-    this.room.setJingleSession(null);
+    this.jingleSession = null;
```

Comment:
i'm just curious, what's the thinking behind moving jinglesession out of room and directly to jitsiconference?